modules/structs/user.go

```diff
@@ -20,16 +20,52 @@ type User struct {
 	FullName string `json:"full_name"`
 	// swagger:strfmt email
 	Email string `json:"email"`
+	// Hide email
+	HideEmail bool `json:"hide_email"`
 	// URL to the user's avatar
 	AvatarURL string `json:"avatar_url"`
+	// URL to the user's API endpoint
+	URL string `json:"url"`
+	// URL to the user's Gitea HTML page
+	HTMLURL string `json:"html_url"`
+	// URL to the user's followers API endpoint
+	FollowersURL string `json:"followers_url"`
+	// URL to the user's following API endpoint
+	FollowingURL string `json:"following_url"`
+	// URL to the user's starred API endpoint
+	StarredURL string `json:"starred_url"`
+	// URL to the user's subscriptions API endpoint
+	SubscriptionsURL string `json:"subscriptions_url"`
+	// URL to the user's organizations API endpoint
+	OrganizationsURL string `json:"organizations_url"`
+	// URL to user's repos API endpoint
+	ReposURL string `json:"repos_url"`
+	// URL to user's heatmap API endpoint
+	HeatmapURL string `json:"heatmap_url"`
+	// Type of the user, User or Org
+	Type string `json:"type"`
+	// Biography about the user
+	Description string `json:"bio"`
+	// Website of the user
+	Website string `json:"website"`
+	// Location of the user
+	Location string `json:"location"`
+	// Public Repo count
+	PubicRepos int64 `json:"public_repos"`
+	// Followers count
+	Followers int `json:"followers"`
+	// Following count
+	Following int `json:"following"`
 	// User locale
 	Language string `json:"language"`
 	// Is the user an administrator
-	IsAdmin bool `json:"is_admin"`
+	IsAdmin bool `json:"site_admin"`
 	// swagger:strfmt date-time
 	LastLogin time.Time `json:"last_login,omitempty"`
 	// swagger:strfmt date-time
-	Created time.Time `json:"created,omitempty"`
+	Created time.Time `json:"created"`
```

Comment:
If the `created` field is empty (I don't know, perhaps an old version of Gitea or some migration), will this break now?